lang/vi.js

```diff
@@ -118,8 +118,8 @@ export default {
 	"Daily Cases": "Daily Cases",
 	"in ICU": "in ICU",
 	"Daily Cases Receiving Treatment": "Daily Cases Receiving Treatment",
-	"Daily Recovered Cases": "Daily Recovered Cases",
-	"of total cases": "of total cases",
+	"Daily Confirmed Cases": "Daily Confirmed Cases",
+  "Per Million": "Per Million",
```

Comment:
@techlab23 Can fix the spacing?

Reply:
@xming13  - Just updated - its weird my editor wasn't showing anything wrong and yet here we are.